enterprise/frontend/src/embedding/data-picker/utils.ts

```diff
@@ -29,13 +20,5 @@ export function getDataTypes({
     dataTypes.push(RAW_DATA_INFO_ITEM);
   }
 
-  if (hasNestedQueriesEnabled && hasSavedQuestions) {
```

Comment:
this function decides what's being shown here.
<img width="394" alt="image" src="https://github.com/user-attachments/assets/8fe96ea9-6428-43d3-a13e-96d3f4d77f8d" />